src/ImageSharp/Configuration.cs

```diff
@@ -83,6 +83,11 @@ public Configuration(params IConfigurationModule[] configurationModules)
         /// </summary>
         public IEnumerable<IImageFormat> ImageFormats => this.imageFormats;
 
+        /// <summary>
+        /// Gets or sets the <see cref="MemoryManager"/> that is currently in use.
+        /// </summary>
+        public MemoryManager MemoryManager { get; set; } = new ArrayPoolMemoryManager(1024 * 80);
```

Comment:
@rytmis `ArrayPool` is actually really useful in small-mid dynamic array allocation use cases, because it reduces GC pressure + improves data locality!
Check out [it's usages in aspnet mvc](https://github.com/aspnet/Mvc/search?utf8=%E2%9C%93&q=JsonOutputFormatter&type=). 

An example: it's [providing](https://github.com/aspnet/Mvc/blob/b196708a3e0bca4820f2002e063fabe979afde1b/src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonOutputFormatter.cs#L48) `ArrayPool<char>` to Newtonsoft.Json [through it's own `IArrayPool<T>` abstraction](https://github.com/JamesNK/Newtonsoft.Json/blob/master/Src/Newtonsoft.Json/JsonTextWriter.cs#L72).

Reply:
I changed the default value to 512 bytes, but obviously you can decide what you want to use for it -- I don't know what benchmarks to run to find the optimal values, and so on, so I'll just defer to your judgment. 🙂